setup.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from setuptools import setup, find_packages
+
+here = os.path.abspath(os.path.dirname(__file__))
+with open(os.path.join(here, 'README.rst')) as f:
+    README = f.read()
+
+with open(os.path.join(here, 'requirements.txt')) as f:
+    requires = filter(None, f.readlines())
+
+setup(
+    name='unicore-hub',
+    version='1.0.0',
```

Comment:
I've found the `VERSION` stuff we're using in other repos to work quite well. Hooking this up with Travis' deploy to pypi stuff ++